reflections/2020-05-22.md

```diff
@@ -0,0 +1,29 @@
+# Reflections
+
+## Participation
+
+- 3 groups 6-8 people in size.
+- Each group had a brainhack committee member and a python/programming enthusiast/expert
+
+## What Happened
+
+- some groups were quiet from the onset and didn't have questions to open up discussion
+- one group had the brainhack member share their screen and run through the lesson
+- another group had questions about what editor they should use (spyder/jupyter notebooks)
+- some participants did not know they could edit their google colab notebooks
```

Comment:
Specifically, they were afraid or unsure if they commented in their colab notebooks, that the changes would affect everyone else? Additionally, they were unsure how to copy, download, and choose the format from options provided for re-use later.